source/jormungandr/jormungandr/default_settings.py

```diff
@@ -109,3 +109,6 @@
     
 GRAPHICAL_ISOCHRONE = False
 HEAT_MAP = False
+# This parameter are used to apply gevent's monkey patch
+# The Goal is to activate parallel calling valhalla, without the patch, parallel http calling may not work
+PATCH_WITH_GEVENT_SOCKET = False
```

Comment:
@pbougue theoretically, this won't affect anything functional but **probably** the performance, which is not handled by artemis